Examples/TodoApp/TodoAppTests/src/data/TaskTests.swift

```diff
@@ -0,0 +1,37 @@
+import XCTest
+@testable import TodoApp
+
+class TaskTests: XCTestCase {
```

Comment:
Equatability should be synthesized for you and therefore does not need to be tested